phoenicis-repository/src/main/java/org/phoenicis/repository/dto/ScriptDTO.java

```diff
@@ -39,6 +39,7 @@
     private final Boolean free;
     private final Boolean requiresPatch;
     private final String script;
+    private final URI icon;
```

Comment:
Can we give this field a more descriptive name? When I use a variable named `icon` I normally expect an `Image` object.

Reply:
It the same as for `ApplicationDTO`. It doesn't make sense to use a different name for both DTOs.